packages/configuration-recommended/index.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I'm going to change the name to `configuration-web-recommended`. In the near future we will have also for development, etc. I think it makes sense.

I'll also add a default configuration for `jsdom`